.github/workflows/test-runner-on-perf.yml

```diff
@@ -1,4 +1,4 @@
-name: Test Runner on Perf Tests
+name: Update Perf Env and Run Nightly Test Runner Tests
 env:
   GOOGLE_APPLICATION_CREDENTIALS: /tmp/jade-dev-account.json
   GOOGLE_SA_CERT: /tmp/jade-dev-account.pem
```

Comment:
Why can we use the /tmp directory for credentials here, but not for integration tests -- is it because of it all being part of the same job or ...? (I think you may have told me this already, so apologies if this is a repeat question.)

In addition, if we're just running Test Runner tests, then I think you should not need the pem file.

Reply:
That's a good question. 
*Most generally?* It's because we're running integration tests in a docker container and the /tmp file isn't mounted.
*Why are we running in a docker container there and not here?* I think it's actually just because we're using our GitHub action repo in integration and that uses a docker container. Whereas in this perf action, we're just running everything from the ubuntu VM in the action. 

I could definitely be wrong here. My original understanding was that this issue stemmed from the fact that we needed to deploy the specified PR's code to the correct int cluster [int-1, int-2, etc]. But, that doesn't explain why we're running into docker-related issues on the client side of things.  In theory, for integration, we should just be able to use the ubuntu VM native with the Github action, and hit the newly deployed to integration cluster, avoiding the /tmp directory issue. 
I'm still slowly piecing together my understanding of all this.

Reply:
And, yes - I'll try ditching the pem file.

Reply:
Looks like the run was successful, so should be good to go!